src/teleop/gui/src/components/AutonTask.vue

```diff
@@ -253,83 +252,39 @@ export default {
 </script>
 
 <style scoped>
-.wrapper {
+.wrap {
   display: grid;
```

Comment:
This is a request for later, but we should start to move away from grid layouts and switch to flexbox. Grid layout actually doesn't get used much anymore in modern webdev and flexbox should fix alot of the issues we have when we resize this viewport.
https://css-tricks.com/snippets/css/a-guide-to-flexbox/